ladybug_geometry/geometry2d/mesh.py

```diff
@@ -290,11 +294,18 @@ def remove_vertices(self, pattern):
                 The length of this list must match the number of this mesh's vertices.
 
         Returns:
-            new_mesh: A mesh where the vertices have been removed according
+            A list with two elements.
+
+            -
+                new_mesh:
+                A mesh where the vertices have been removed according
```

Comment:
This looks a little awkward here.  Is there any way we could avoid the extra line break between the bullet point and the text that follows it?

Reply:
Fixed! I figured out we didn't need the line break, was the tab spacing all it needed.